masschallenge/impact-api
# AC-5700: allocate_applications API call

** BE SURE TO MERGE https://github.com/masschallenge/django-accelerator/pull/77 IN django-accelerator BEFORE THIS **
(or immediately after if this happens to get merged first)

To test:
- Be sure to get the AC-5700 branch from django-accelerator
- Load a special database that gets a judging round setup properly.  E.g., `make load-db s3_key=AC-5700.sql.gz`.  You are expected to get warnings about "changes that are not yet reflected in a migration" that can be safely ignored.
- Start both the accelerate server and the impact-api server.
- Login to your local accelerate server as a confirmed judge for an active judging round (e.g., 291-user@example.com for the PULSE 2018 Round 1 (id=56))
- You should see the /panels page with a message at the bottom that says "Thank You! Can you review more?"
- Still logged in as the judge go to http://localhost:8000/api/v1/allocate_applications/56/291/
- This should return a list of application ids that they have been assigned.
- Reload the /panels page and see the assigned applications.
- If you run http://localhost:8000/api/v1/allocate_applications/56/291/ again it should give an error.
- Extra credit for the interested developer: Go through the review process for this judge, e.g. mark everything as "not judged conflict".  Mark the panel as done.
- You should now get a new set of application if you run http://localhost:8000/api/v1/allocate_applications/56/291/ again.
